AMW_angular/io/src/app/apps/apps-list/apps-list.component.html

```diff
@@ -0,0 +1,16 @@
+@if (apps && apps.length > 0) {
+<table class="table table-sm table-borderless mb-0">
+  <tbody>
+    @for (app of apps; track app) {
+    <tr [class.table-light]="!even">
```

Comment:
I don't understand this. `even` is a property that gets passed into the component - how does it change when iterating over list of apps?

Reply:
should probably be 
`@for (app of apps; track app; let even = $even) {`

and then remove to input property from the component

Reply:
ouch - this is a nested table... then the input property makes sense.

But nesting tables seems to be a bad idea. Is this even valid html?